src/components/manage/Blocks/Image/View.jsx

```diff
@@ -32,15 +36,26 @@ export const View = ({ data, detached }) => {
         <>
           {(() => {
             const image = (
-              <img
-                className={cx({
+              <Image
+                className={cx('responsive', {
                   'full-width': data.align === 'full',
                   large: data.size === 'l',
                   medium: data.size === 'm',
                   small: data.size === 's',
                 })}
+                item={
+                  data.image_scales
+                    ? {
+                        '@id': data.url,
+                        image_field: data.image_field,
+                        image_scales: data.image_scales,
+                      }
+                    : undefined
+                }
                 src={
-                  isInternalURL(data.url)
+                  data.image_scales
+                    ? undefined
```

Comment:
I appreciate the use of srcset and further modern image options. But…
Image block images aligend left or right and especially of smaller size than 'large' are fetched in viewport size. That's much more than necessary. 
Let's think about a way to give the browser a hint to fetch the appropriate size.

To reproduce the situation that causes my headache:
- Create an image block and upload a bigger than big image.
- Align left
- Size it to medium

![Battery_and_NowPlaying_and_Item-0_and_Item-0_and_Item-0_and_AppleTimeMachineExtra-2](https://github.com/plone/volto/assets/1144737/160191b4-068b-48cc-be8a-3fa74c6f6f60)

Reply:
> I appreciate the use of srcset and further modern image options. But… Image block images aligend left or right and especially of smaller size than 'large' are fetched in viewport size. That's much more than necessary. Let's think about a way to give the browser a hint to fetch the appropriate size.
> 
> To reproduce the situation that causes my headache:
> 
>     * Create an image block and upload a bigger than big image.
> 
>     * Align left
> 
>     * Size it to medium
> 
> 
> ![Battery_and_NowPlaying_and_Item-0_and_Item-0_and_Item-0_and_AppleTimeMachineExtra-2](https://user-images.githubusercontent.com/1144737/248552181-160191b4-068b-48cc-be8a-3fa74c6f6f60.png)

before on 17a14

![Cursor_and_image_rendering](https://github.com/plone/volto/assets/1144737/6e9a962e-f708-414a-9b0c-d1f8cc83eaaf)

Reply:
Yeah you are right, I even documented how to do this ad then did not do it here :) I will fix this

Reply:
@ksuess added the sizes prop to the image and leadimage blocks. Should now work way better, it does for me. Using this approach also considers the pixel density of the screen, which is a nice improvement.

Reply:
@pnicolli great solution!